.github/workflows/run_benchmark_multi_table.yaml

```diff
@@ -0,0 +1,60 @@
+name: Run SDGym Benchmark Multi-Table
```

Comment:
if it's not too much work, you could refactor these workflows into 3:
- one callable or manually triggered workflow that runs on the modality you specify as input
- a scheduled multi-table one that calls the one above
- a scheduled single table one that calls the one above